coverage/lcov-report/data/exploreInfo.ts.html

```diff
@@ -0,0 +1,178 @@
+
```

Comment:
Delete this card (Explore the basics) entirely. Text has been moved to "About Tigera products"